src/bci_build/package/__init__.py

```diff
@@ -589,22 +594,8 @@ def build_version(self) -> str | None:
     def build_release(self) -> str | None:
         if self.os_version not in (OsVersion.SP6,):
             return None
-        if "dotnet" in self.build_name:
-            if "8.0" in self.build_name:
-                return "20"
-            if "6.0" in self.build_name:
-                return "32"
-        if "golang" in self.build_name:
-            return "33"
-        if "python" in self.build_name:
-            return "41"
-        if "php" in self.build_name:
-            return "30"
-        if "node" in self.build_name:
-            return "30"
-        if isinstance(self, ApplicationStackContainer):
-            return "35"
-        return None
+
+        return str(self._build_release) if self._build_release is not None else None
```

Comment:
well, the values are actually SP specific, so they would have to be different/higher for SP7. that's why I opted for the less clean but more central approach to map those numbers rather than having them as part of the attributes of the containers.